event-session-signup/README.md

```diff
@@ -1,12 +1,26 @@
 # Sign Up for Sessions at an Event
 
-Allows event attendees to sign up for sessions at an event and creates an itinerary.
+_Allows event attendees to sign up for sessions at an event and creates an itinerary._
 
-[Read more](https://developers.google.com/gsuite/solutions/event-session-signup)
+Last updated: July, 2019
 
-## Set Up Instructions
+Quickly create an end to end conference registration system. Start with the
+conference event information listed in a Google Sheet, then use Apps Script to
+create a sign-up form. For each attendee, a personalized schedule in a Google
+Doc is emailed out and a conference calendar is created then populated with the
+specific sessions.
 
-<!-- [START setup] -->
+![demo](https://cdn.jsdelivr.net/gh/gsuitedevs/solutions@master/event-signup/event-signup.jpg)
+
+## Technology highlights
+
+- Create a custom menu item in the Spreadsheet UI using Apps Script [Custom Menus](/apps-script/guides/menus).
```

Comment:
Can you explain where the custom menu item is? :)

Reply:
Good catch, you'll need to convert all the relative URLs into absolute ones.

Reply:
Fixed and verified all links